### ISSUE TITLE
ReferenceError: Can't find variable: inspect

### ISSUE THREAD

**Issue by aj0strow** [bySabi/karma-tap#7]
Hey was hoping to use your plugin but it seems to break on an undeclared `inspect` function. If you have any ideas how to fix, i would greatly appreciate it. 

https://github.com/mapbox/karma-tap/blob/master/src/adapter.js#L39

```
 ReferenceError: Can't find variable: inspect
  at ./node_modules/karma-tap/src/adapter.js:39
```

**Comment by aj0strow** [comment]
I took a random guess that it was supposed to be `util.inspect` and replace with:

``` js
Object.toString.call(args[i])
```

It still spits tap output to stdout

```
PhantomJS 1.9.8 (Mac OS X 0.0.0) LOG: 'ok 16 should be equal'
```

However up and running. Hope this helps.

**Comment by anandthakker** [comment]
@tmcw Could you cut an npm release to include this fix?

**Comment by brandonjpierce** [comment]
@tmcw +1 to updating the NPM release as well.

**Comment by facundocabrera** [comment]
hey! I'm having the same issue, could you please publish the fix in npm?

**Comment by tmcw** [comment]
Published as v1.0.4

This module is up for adoption :) If you use it, consider adopting it! I don't use it anymore, and have no moral compass for changes to it.

**Comment by facundocabrera** [comment]
@tmcw thanks a lot for your time releasing the package :smile: 
Is there any plan to merge with another project? (I didn't look for another solution, this pkg is just what I need)
